code-challenges.js

```diff
@@ -39,3 +104,24 @@ var numbersToAdd2 = [0, 7, -8, 12]
 // Expected output: [0, 7, -1, 11]
 var numbersToAdd3 = []
 // Expected output: []
+
+
+//Creation of function that will takes in an array and returns an array of the accumulating sum
+const accumulate = (array) =>{
+    //Declaring a temp variable to hold the index data and do the actual calc, but for now, it just sits around looking gorgeous. Just.. dont be rude and let it know its a 0 right now. 
+    let temp = 0;
+    //Declaring a temp array that holds nothing, but will eventually have above calculation pushed into it. Dont tell it that its just being used. Please. It cant handle it. Very sensitive.
+    let tempArray = []
+    //For loop to do the mafs. 
+    for (let i = 0; i<array.length; i++){
+        temp = array[i] + tempArray
+        tempArray.push(temp)
+    }
+    //Returning the logic! Yay! temp and tempArray can feel useful! 
+    return tempArray
```

Comment:
the return here isn't the expected output. right now, for numbersToAdd1 you get `[ '2', '42', '452,42', '92,42,452,42' ]`. I think this comes from trying to add `array[i]` + `tempArray`. I don't think it's doing quite what you expect